subprojects/docs/src/docs/release/notes.md

```diff
@@ -53,6 +53,8 @@ The following are the newly deprecated items in this Gradle release. If you have
 ### Example deprecation
 -->
 
+* `Task.deleteAllActions()` is deprecated without replacement.
```

Comment:
Do we need a header here or is a list like you started sufficient?

Reply:
We could add something like "Other minor deprecations" but if this is the only one in the end, then we will need to rewrite it anyway. I'd leave it like this until before the release when we release notes anyway. The important thing is to capture the deprecation now.